src/core/client/stream/tabs/Comments/Comment/InReplyTo.tsx

```diff
@@ -1,22 +1,55 @@
 import { Localized } from "@fluent/react/compat";
 import cn from "classnames";
-import React, { FunctionComponent } from "react";
+import React, { FunctionComponent, useCallback } from "react";
 
+import { useCoralContext } from "coral-framework/lib/bootstrap";
+import { globalErrorReporter } from "coral-framework/lib/errors";
 import CLASSES from "coral-stream/classes";
+import computeCommentElementID from "coral-stream/tabs/Comments/Comment/computeCommentElementID";
 import { Flex, Icon } from "coral-ui/components/v2";
 
+import { CommentContainer_comment as CommentData } from "coral-stream/__generated__/CommentContainer_comment.graphql";
+
 import styles from "./InReplyTo.css";
 
 interface Props {
-  username: string;
+  parent: CommentData["parent"];
+  enableJumpToParent: boolean;
 }
 
-const InReplyTo: FunctionComponent<Props> = ({ username }) => {
-  const Username = () => (
-    <span className={cn(styles.username, CLASSES.comment.inReplyTo.username)}>
-      {username}
-    </span>
-  );
+const InReplyTo: FunctionComponent<Props> = ({
+  parent,
+  enableJumpToParent,
+}) => {
+  const { pym } = useCoralContext();
+
+  const navigateToParent = useCallback(() => {
+    if (!parent) {
+      return;
+    }
+    const elemId = computeCommentElementID(parent.id);
+    const elem = document.getElementById(elemId);
+    if (elem) {
+      void pym?.scrollParentToChildEl(elemId);
+      elem.focus();
```

Comment:
ah, yes, that's what I forgot in my previous comment, to use pym to facilitate scrolling. Good job here!